config/locales/fr.yml

```diff
@@ -669,6 +669,12 @@ fr:
       view_more_samples: Voir plus d'échantillons
     puid:
       title: 'Copy to clipboard value: %{puid}'
+    samples:
+      table_component:
+        counts:
+          status:
+            one: "%{selected} sur %{total} échantillon sélectionné"
```

Comment:
Same comment as above. This can probably be changed to `1 sur 1 échantillon sélectionné`